### ISSUE TITLE
Failing to load scss resource even after deleting the file and rebuilding application

### ISSUE THREAD

**Issue by rigorio** [ionic-team/ionic#17486]
# Bug Report

All was working well. I had a page generated with `ionic g page reservations`.
I already wrote on `reservations.html` and `reservations.ts`, it was really working as I was able to navigate to it and browse it via `ionic serve` in my desktop browser.

Then I created a class for an element in `reservations.html` and added the class to the `reservations.scss`.
That's when this happened:

```
GET http://localhost:8100/reservations/reservations.scss 404 (Not Found)
Unhandled Promise rejection: Failed to load ../reservations/reservations.scss ; Zone: <root> ; Task: Promise.then ; Value: Failed to load ../reservations/reservations.scss
```

I've tried clearing my cache (what I read on some forums). I'm on Brave browser (chrome based). Nothing happened.

Here's what I've tried so far:
1. Delete the entire folder for `reservations`
2. Try to run the app
3. Surprisingly, even after deleting that directory where `reservations.scss` was in, the results were the same. It was still looking for `reservations.scss`. 
4. `ionic g page reservations`
5. Delete `reservations.module.ts` as I usually do as I don't need it (I do this with all of my generated pages, never encountered a problem before)
6. Add ReservationsPage to `app.module.ts`
7. Run the app, same error 

I've also tried `ionic build` and `ionic build browser`. No cigar

IIRC, I'm on Ionic v 3.9.x. Sorry I can't provide the exact version, because apparently when I type `ionic.version` on the browser console, it now gives me this error:

```
Uncaught ReferenceError: ionic is not defined
    at <anonymous>:1:1
```

I would highly appreciate it if someone were to help me out immediately, I'm honestly trying to meet a deadline, so this came at the worse possible time. 😢

**Comment by rigorio** [comment]
Update: 
I deleted the `reservations` directory again and removed it from `app.module.ts` and the one ts file it was being called upon.
Then I did (not that I know what I'm doing at this point it's already midnight where I am):
1. ionic build
2. ionic build browser
3. npm install

Then I tried to check where exactly it's trying to look for reservations.scss. I'm on intellij, so `shift + shift` opens the search everywhere thing and I search for it, nothing shows up. Seems alright.

And now I run the app again, same error. Still looking for it.

Also tried on different browsers (Mozilla and Google Chrome), same results.

**Comment by KillerCodeMonkey** [comment]
if you are on ionic v3 you should create the issue there https://github.com/ionic-team/ionic-v3.

sounds like something is wrong with your build, because after build there should be css files and not scss files.

Could you share your project folder/file structure?

And keep in mind in ionic v3 you do not need to add your .scss file in the `@Component` decorator.
It is enough when the .scss file is named like the page file.

**Comment by rigorio** [comment]
Sorry for the late reply I was at my daytime work. I've looked into it further..

Turns out, the problem wasn't Ionic - Intellij Webstorm was the culprit. I'm a user that likes to utilize the refactoring (renaming) tool _a lot_. Something _may_ have happened when I was renaming this or that, Webstorm messed it up, but now it's all good. 

I'll be closing this now. Sorry for the false alarm! And thank you for your quick reply!
